objects-api/src/main/kotlin/com/ritense/objectsapi/taak/resolve/ProcessVariableValueResolver.kt

```diff
@@ -0,0 +1,16 @@
+package com.ritense.objectsapi.taak.resolve
```

Comment:
We might want an additional PlaceHolderValueResolver. 
Because now we have one for `pv:` and one for `doc:`. But none for just a static variable